src/Domain/FundType.php

```diff
@@ -37,4 +37,9 @@ public function allocationOrder(): int
         // check tests cover match above.
         return $order * 100;
     }
+
+    public function isPledge(): bool
+    {
+        return $this === self::Pledge || $this === self::TopupPledge;
```

Comment:
I think it's worth using match so we can't forget to update this if we ever have any other type of pledge:
```suggestion
        return match ($this) {
            self::Pledge, self::TopupPledge => true,
            self::ChampionFund => false,
        };
```